Analysis/Core/include/Analysis/TrackSelection.h

```diff
@@ -26,6 +26,26 @@ class TrackSelection
  public:
   TrackSelection() = default;
 
+  enum TrackCuts {
```

Comment:
maybe we could use here an 'enum class' to make it more save (later in the code you anyways do the cast to the enum type).
TrackSelection.h will probably be included in all analyses and we should avoid polluting the namespaces of all tasks with these symbols (e.g. it might be that somebody wants to use kDCAz for some other purpose in his analysis)